### ISSUE TITLE
2.5.2 and CHIP

### ISSUE THREAD

**Issue by sanc909** [pi-hole/pi-hole#322]
Should 2.5.2 run on a CHIP running debian? I have the old version but if I try to run the installer it days there's insufficient disk space. It suggests running raspi-config...

Initating network interface                                                  Insufficient Disk space

```
chip@chip:~$ df -h                                                          
Filesystem      Size  Used Avail Use% Mounted on                             
ubi0:rootfs     3.7G  478M  3.2G  13% /dev
tmpfs        248M     0  248M   0% /dev                                   
tmpfs           248M     0  248M   0% /dev/shm                               
tmpfs           248M   15M  234M   6% /run                                   
tmpfs           5.0M  4.0K  5.0M   1% /run/lock                              
tmpfs           248M     0  248M   0% /sys/fs/cgroup                         
tmpfs            50M     0   50M   0% /run/user/1000                         
chip@chip:~$
```

**Comment by jacobsalmela** [comment]
I have a C.H.I.P. and another one coming in the mail today.  I'll have to test this out.  Last time I installed it, it went fine, but we have added a lot more stuff...

**Comment by CDeeRON** [comment]
i updated the pihole install on my CHIP yesterday and had to modify the install script to check “/dev” instead of “/“. Afterwards the install completed successfully.

**Comment by PromoFaux** [comment]
We have added the 25MB check. Sounds like its hitting that. Do you need to expand your file system? Or does it work slightly differently on a CHIP? (Note to self.. Really must get my hands on one!)

**Comment by sanc909** [comment]
Many Thanks CDeeRon. Changing the test from / to /dev gives a value. I ran the test in a terminal and you can see that testing / gives "df: no file systems processed"  but testing /dev gives a value. 
```
chip@chip:~$ df -h
Filesystem      Size  Used Avail Use% Mounted on
ubi0:rootfs     3.7G  478M  3.2G  13% /
devtmpfs        248M     0  248M   0% /dev
tmpfs           248M     0  248M   0% /dev/shm
tmpfs           248M   15M  234M   6% /run
tmpfs           5.0M  4.0K  5.0M   1% /run/lock
tmpfs           248M     0  248M   0% /sys/fs/cgroup
tmpfs            50M     0   50M   0% /run/user/1000
chip@chip:~$ # this is the old test
chip@chip:~$ df -lkP / | awk '{print $4}' | tail -1
df: no file systems processed
chip@chip:~$ # this is the new test
chip@chip:~$ df -lkP /dev | awk '{print $4}' | tail -1
253292
chip@chip:~$
```

I changed the code and it ran fine.

**Comment by jacobsalmela** [comment]
We can probably throw in some logic under the 25MB check to see if the user is on a C.H.I.P. with `uname -a` and adjust the code accordingly.

**Comment by sanc909** [comment]
Many thanks for the prompt responses and assistance. Installed completed
and all is well.

One other thing I've noticed. On the webpage If I click on the more info
link nothing happens. Is that the correct behaviour?

**Comment by CDeeRON** [comment]
I would use the following which will just pull the first local filesystem(always ROOT FS) output from df. This should fix for CHIP or any other flavor of unix.

#df -lk | awk '{print $4}' | head -2 | tail -1

**Comment by jacobsalmela** [comment]
Excellent.  We'll give it a try.

**Comment by Mcat12** [comment]
@sanc909 there currently is no behavior associated with the more info buttons. There will be a big update coming when we get it stable enough.

**Comment by sanc909** [comment]
Ah. Thanks for the reply. Just wanted to check it wasn't a CHIP quirk.

On Tue, Feb 16, 2016 at 8:25 pm, Mcat12 < notifications@github.com [notifications@github.com] > wrote:
@sanc909 [https://github.com/sanc909] there currently is no behavior associated with the more info buttons. There
will be a big update coming when we get it stable enough.

—
Reply to this email directly or view it on GitHub
[https://github.com/pi-hole/pi-hole/issues/322#issuecomment-184862701] .[https://github.com/notifications/beacon/AOoLQ3XMqMI1Hr17oUaEj62DpC7JEMxCks5pk309gaJpZM4HbcZB.gif]

**Comment by PromoFaux** [comment]
@CDeeRON I may be being dumb here.. what is the difference between what you suggest, [and this line here](https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh#L123) ?

**Comment by PromoFaux** [comment]
Never mind. It's the `/`

silly me!

**Comment by CDeeRON** [comment]
No worries. My thought with the cmd is output all local filesystems give me 4th column then parse the available kbytes for first filesystem. This would provide available kbytes regardless if ROOTFS is mounted say /root, / or some other variation or flavor of unix.

**Comment by PromoFaux** [comment]
That makes sense. And is a harmless enough change. Thanks for your contribution.

**Comment by PromoFaux** [comment]
@jacobsalmela If you get a chance, could you latest devlopment branch version on your C.H.I.P, please?

**Comment by jacobsalmela** [comment]
This will be fixed by #330

**Comment by DonLexos** [comment]
This has been fixed in release 2.5.3
